pkg/ansible/controller/reconcile.go

```diff
@@ -183,7 +183,12 @@ func (r *AnsibleOperatorReconciler) Reconcile(request reconcile.Request) (reconc
 			}
 		}
 		if event.Event == eventapi.EventRunnerOnFailed && !event.IgnoreError() {
-			failureMessages = append(failureMessages, event.GetFailedPlaybookMessage())
+			if len(event.GetFailedPlaybookMessage()) < 1 {
```

Comment:
I'm not sure I follow what this block is doing, are we actually hitting this branch and if so does it matter if `event.GetFailedPlaybookMessage()` is `""`?

Reply:
Yes, it matters.
If the event.GetFailedPlaybookMessage() is "" then, `runSuccessful := len(failureMessages) == 0` besides has an error.

Reply:
that's strange, wouldn't failure messages then be `[]string{""}`, which should still have a length of 1 (https://play.golang.org/p/ZcvpUbPIMct)?

Reply:
You are right `runSuccessful := len(failureMessages) == 0` will work well since it will be 1 in the scenario of "". However, it will have no msg. Because of this, I added it.